.github/CONTRIBUTING.md

```diff
@@ -88,12 +87,15 @@ issues of each type.
 
 #### Questions
 
- - [ ] __Search for answers in Terraform documentation__: We're happy to answer
-   questions in GitHub Issues, but it helps reduce issue churn and maintainer
-   workload if you work to find answers to common questions in the
-   documentation. Often times Question issues result in documentation updates
-   to help future users, so if you don't find an answer, you can give us
-   pointers for where you'd expect to see it in the docs.
+Please do not use GitHub to ask questions! Instead:
+
+ - [ ] __Search for answers in Terraform documentation__
```

Comment:
The use of the checklist syntax feels weird here (both in the old and new versions).

I guess it was just for consistency with the other lists below? I guess it's only cosmetic anyway, but it feels strange to include checkboxes for a list of the form "choose one of these" as opposed to "complete all of the following". :thinking:

Reply:
Yeah, I agree that it's awkward, I'll make it bullet points instead. I don't like it either :)